src/sentry/consumer.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import, print_function
+
+from batching_kafka_consumer import AbstractBatchWorker
+
+from django.conf import settings
+
+import sentry.tasks.store as store_tasks
+from sentry.utils import json
+
+
+def handle_preprocess(message):
+    data = message['data']
+    event_id = data['event_id']
+    cache_key = message['attachments_cache_key']
+    start_time = message['start_time']
+    process_task = (
+        store_tasks.process_event_from_reprocessing
+        if message['from_reprocessing']
+        else store_tasks.process_event
+    )
+
+    store_tasks._do_preprocess_event(cache_key, data, start_time, event_id, process_task)
+
+
+def handle_process(message):
```

Comment:
Any reason why these aren't class methods on `ConsumerWorker` ?

Reply:
Oh, nope, I think that was a partial refactor that I didn't need, I'll move them.